CMakeLists.txt

```
@@ -162,10 +170,17 @@ set(PT_LIBS
 )
 
 if (${TRITON_PYTORCH_ENABLE_TORCHVISION})
-  set(PT_LIBS
-      ${PT_LIBS}
-      "libtorchvision.so.1"
-  )
+  if(${RHEL_BUILD})
+    set(PT_LIBS
+        ${PT_LIBS}
+        "libtorchvision.so"
+    )
+  else()
+    set(PT_LIBS
+        ${PT_LIBS}
+        "libtorchvision.so.1"
+    )
+  endif()
```

Using [generator-expressions](https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html) is more preferable in my opinion: 
```suggestion
  set(PT_LIBS
         ${PT_LIBS}
         $<IF:$<BOOL:RHEL_BUILD>,libtorchvision.so,libtorchvision.so.1>
  )
```
see: https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#conditional-expressions